ui/src/vayla-design-lib/datepicker/datepicker.tsx

```diff
@@ -35,7 +36,7 @@ function getHeaderElement({
                 />
             </div>
             <span className={styles['datepicker__current-month']}>
-                {format(date, 'MMMM')} {date.getFullYear()}
+                {format(date, 'MMM', { locale: fi })} {date.getFullYear()}
```

Comment:
Tässä myös pieni toinekin muutos. Vanhempi "MMMM" kääntyy "Heinäkuuta", kun taas "MMM" kääntyy "Heinä". Tuo jälkimmäinen vaikutti ainakin omaan silmään siistimmältä ja vähemmän tönköltä

Reply:
Kokeilitko LLLL:ää? Sillä pitäis tulla pelkkä "Heinäkuu"